axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/PurchaseOrderWorkflowServiceSupplychainImpl.java

```diff
@@ -86,6 +89,8 @@ public void validatePurchaseOrder(PurchaseOrder purchaseOrder) throws AxelorExce
   @Override
   @Transactional
```

Comment:
Please fix the Transactionnal annotation, it could not rollback when an AxelorException is thrown here